src/main/java/org/jitsi/videobridge/octo/OctoEndpoint.java

```diff
@@ -91,6 +92,12 @@ public void onNewSsrcAssociation(
 
     }
 
+    @Override
+    public void sendRtp(PacketInfo packet, String sourceEpId)
```

Comment:
Should we move the PotentialPacketHandler implementation to Endpoint?

Reply:
so then we'd need to split it into 2 interfaces, right?  one for `wants`, which would be at `AbstractEndpoint` and another for `sendRtp` which `Endpoint` and `OctoTentacle` would implement?